config/v1/types_node.go

```diff
@@ -70,6 +75,17 @@ const (
 	DefaultUpdateDefaultReaction WorkerLatencyProfileType = "Default"
 )
 
+// +kubebuilder:validation:Enum=Enabled;Disabled;""
```

Comment:
If you are using `omitempty` which I think you are doing because this is a workload API rather than a configuration API, you don't need to have `""` in place, when the field is omitted the validation for the enum will not execute

Reply:
Is it a singleton resource or is there one per Node in the cluster? If it's a singleton then yep, it's a configuration API, in which case I would suggest including `""` in the enum and dropping `omitempty`. This improves the discoverability of the API for the user to configure it

Reply:
Agreed with your point, Updated !!